DOCUMENTATION.md

```diff
@@ -0,0 +1,107 @@
+# Armbian Documentation
+
+[![Create offline documentation to release](https://github.com/armbian/documentation/actions/workflows/release.yaml/badge.svg)](https://github.com/armbian/documentation/actions/workflows/release.yaml)
+
+<p align="center">
```

Comment:
I remember years ago someone mentioned that markdown docs shall not contain html? @lanefu maybe?

Reply:
We have this header on all README.md ... so changing all or none.

Reply:
Well can be addressed later as well.